ports/bext-ut/portfile.cmake

```
@@ -0,0 +1,29 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO boost-ext/ut
+    REF v1.1.8
+    SHA512 0df2f8ce784dc69c3cca5554a8b2be7c1664dd66bf52e83f305db5fae84d416a851e9282e9a8cfe50fbcada85b17da00fb25c1228d9cca32226a18bae18acb83
+    HEAD_REF master
+)
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+    PREFER_NINJA
+    OPTIONS
+        -DBOOST_UT_BUILD_BENCHMARKS=OFF
+        -DBOOST_UT_BUILD_EXAMPLES=OFF
+        -DBOOST_UT_BUILD_TESTS=OFF
+)
+vcpkg_install_cmake()
+vcpkg_copy_pdbs()
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/ut)

```

```suggestion
vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/ut TARGET_PATH share/ut)
```
I assume that the expected call is `find_package(ut CONFIG REQUIRED)`? This will currently create `share/bext-ut/...`.

---

```
bext-ut:/include/boost/ut.hpp
bext-ut:/share/bext-ut/copyright
bext-ut:/share/bext-ut/ut-config.cmake
bext-ut:/share/bext-ut/vcpkg_abi_info.txt
```